src/models.ts

```diff
@@ -32,3 +70,70 @@ export const baseMetricValues: Record<BaseMetric, BaseMetricValue[]> = {
   [BaseMetric.INTEGRITY]: ['N', 'L', 'H'],
   [BaseMetric.AVAILABILITY]: ['N', 'L', 'H']
 };
+
+export const environmentalMetricValues: MetricValues<
+  EnvironmentalMetric,
+  EnvironmentalMetricValue
+> = {
+  [EnvironmentalMetric.ATTACK_VECTOR]: ['N', 'A', 'L', 'P', 'X'],
+  [EnvironmentalMetric.ATTACK_COMPLEXITY]: ['L', 'H', 'X'],
+  [EnvironmentalMetric.PRIVILEGES_REQUIRED]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.USER_INTERACTION]: ['N', 'R', 'X'],
+  [EnvironmentalMetric.SCOPE]: ['U', 'C', 'X'],
+  [EnvironmentalMetric.CONFIDENTIALITY]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.INTEGRITY]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.AVAILABILITY]: ['N', 'L', 'H', 'X'],
+  [EnvironmentalMetric.CONFIDENTIALITY_REQUIREMENT]: ['M', 'L', 'H', 'X'],
+  [EnvironmentalMetric.INTEGRITY_REQUIREMENT]: ['M', 'L', 'H', 'X'],
+  [EnvironmentalMetric.AVAILABILITY_REQUIREMENT]: ['M', 'L', 'H', 'X']
+};
+
+export const temporalMetricValues: MetricValues<
```

Comment:
please move it above `environmentalMetricValues` to have unified order (base, temporal, environmental) everywhere